src/ProjectTemplates/Web.ProjectTemplates/content/EmptyComponentsWebAssembly-CSharp/.template.config/ide.host.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json.schemastore.org/vs-2017.3.host",
+  "order": 660,
+  "icon": "icon.png",
+  "ports": [
+    {
+      "name": "HttpPort",
+      "useHttps": false
+    },
+    {
+      "name": "HttpsPort",
+      "useHttps": true
+    }
+  ],
+  "disableHttpsSymbol": "NoHttps",
+  "symbolInfo": [
+    {
+      "id": "Hosted",
+      "isVisible": "true"
```

Comment:
Do we want these parameters to be "sticky"?
if so, add the following to each symbolInfo in ide.host.json. This will persist the last choice chosen in this template for the next time it is selected in Visual Studio

`      "persistenceScope": "templateGroup"`